File: program/program/convert-string-to-constantcase/ConvertStringToConstantcase.java
import java.util.Scanner;

public class ConvertStringToConstantcase {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String a = sc.nextLine();
        System.out.print("Input: ");
        String b = a.replace(' ', '_');
        System.out.println(b.toUpperCase());

        sc.close();
    }
}

File: program/program/convert-string-to-constantcase/ConvertStringToConstantcase.kt
import java.util.Scanner

fun main() {
    print("Enter String : ")
    println()
    var min = readLine()

    println("Constant Case: " + ConvertStringToConstantcase(min!!.split(" ")))

}

private fun ConvertStringToConstantcase(args: List<String>): String {
    val buffer = StringBuffer()
    for((index,day) in args.withIndex()){
        //println("$index :$day")
        if(index == 0) {
            buffer.append(day.uppercase());
        } else {
            buffer.append("_").append(day.uppercase())
        }
    }

    return buffer.toString()
}